src/app/[locale]/(pages)/ripristino-accesso/page.tsx

```diff
@@ -25,12 +25,14 @@ const RestoreProfile = (): React.ReactElement => {
         <Grid container flexDirection={'column'}>
           <IdpListOnApp />
           <Grid item sm={10} md={7}>
+            {/* IF SPID level from token is L3 hide Typography line 29-33 */}
```

Comment:
I wouldn't put a reference on the line number, because it could change and we should forget to update it.

Reply:
Ok, i will remove the line number on comment.